### PR TITLE
Update avenga-git-guidelines.md

### LINKED ISSUE #10: Clean commits? I beg to differ.
This issue relates to one of the [Git Guidelines](https://github.com/avenga/coding-guidelines/blob/master/general/avenga-git-guidelines.md#commits) which I would like to open for discussion. Namely this one:

>### Commits
>* **Clean commits:** Commits must only contain changes related to the task/feature you are working on.

&nbsp;
I'm a firm believer to further slow down code rot and to combat "broken window theory" type coding ([as visualized here](https://www.monkeyuser.com/2019/code-entropy/)), any programmer should develop an "Always leave the campground cleaner than you found it." mindset.
[Uncle Bob also wrote this in his "Clean Code"](https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)

To make my point, I feel this "clean commits" rule kind of prohibits cleaning up the campground even if my task is to add some new feature. Adding small code cleanups or refactorings to my PRs should be encouraged, not complicated for the dev. If I have to create a new branch and open a separate PR for miniscule changes it would discourage me from doing it in the first place.  

If, by any chance, this rule only specifies that one should create separate commits for changes not related to the current task (which I would support 100%), I would at least like to change the wording of it.

### PR DESCRIPTION
**Pull request by b-boehne**
Closes #10

### CODE REVIEW

**Review by frederik-b — APPROVED** [review]

File: general/avenga-git-guidelines.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Commits
 * **JIRA ticket code:** Always add JIRA ticket code wrapped in square brackets to the end of your commit message, example: "CoreBase: Redesign Lists page **[CB-75]**".
 * **Sub-project's name prefix:** If project consists of several sub-projects like iOS app, Android app, web app, etc, use sub-project's name as a prefix of commit message, for example: "**Android:** Add login screen [CB-101]", "**iOS:** Time reporting screen redesign [CB-102]", "**CoreBase:** Redesign phase 1 [CB-103]", etc.
-* **Clean commits:** Commits must only contain changes related to the task/feature you are working on.
+* **Clean commits:** Changes not related to the task/feature you are working on should be included in separate commits.
```
Comment (GrzegorzMachnik) [inline general/avenga-git-guidelines.md]: Changes not related to the task/feature you are working on (code clean up, refactor, typo fixes etc.) should be included in separate commits.

**Review by eaderhold — APPROVED** [review]
Thanks for the PR! Much better!

File: general/avenga-git-guidelines.md (hunk 1 above)
Comment (b-boehne) [reply general/avenga-git-guidelines.md]: Sure, some more clarification makes sense here.

**Review by GrzegorzMachnik — APPROVED** [review]
